docs/Concepts/Pruning.md

```diff
@@ -0,0 +1,13 @@
+description: Pruning
+<!--- END of page meta data -->
+
+# Pruning
+
+Use pruning on Mainnet to remove state tries that aren't necessary. Otherwise, the state will continue to grow, and storage will become a problem.
+
+!!! Caution
+    Do not use pruning if you plan to make [private transactions](Privacy/Privacy-Overview.md).
+
+Before you enable pruning, ensure you have at least 600 GB of available space. To toggle pruning enablement, use the [`--pruning-enabled`](../Reference/CLI/CLI-Syntax.md#pruning-enabled) option.
```

Comment:
The first sentence isn't quite right for a couple of reasons: 
* The 600 GB of space relates to mainnet only.  ie, you could use pruning on Ropsten and the space requirement would be different. 
* Having enough space is a requirement for synchronizing the chain - the way this is worded makes it sound like space is required because of pruning which is misleading because the point of pruning is to reduce the storage required.

Reply:
To toggle pruning enablement -> To enable pruning 

Suggested edit to both be more active and consistent with the rest of the doc

Reply:
Updated